testing/btest/Baseline/scripts.base.frameworks.logging.telemetry/telemetry.log

```diff
@@ -10,7 +10,7 @@
 XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_stream_writes_total	module,stream	Conn,Conn::LOG	34.0
 XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_stream_writes_total	module,stream	DNS,DNS::LOG	34.0
 XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_stream_writes_total	module,stream	HTTP,HTTP::LOG	14.0
-XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_writer_writes_total	writer,module,stream,filter-name,path	default,Conn,conn,Conn::LOG,Log::WRITER_ASCII	30.0
-XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_writer_writes_total	writer,module,stream,filter-name,path	default,DNS,dns,DNS::LOG,Log::WRITER_ASCII	23.0
-XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_writer_writes_total	writer,module,stream,filter-name,path	default,HTTP,http,HTTP::LOG,Log::WRITER_ASCII	10.0
+XXXXXXXXXX.XXXXXX	zeek	counter	zeek_log_writer_writes_total	writer,module,stream,filter-name,path	Log::WRITER_ASCII,Conn,Conn::LOG,-,conn	30.0
```

Comment:
Heh. So we did test it, we just missed it.

Reply:
Yeah... 😬